pkg/clusteragent/admission/mutate/common.go

```diff
@@ -222,46 +216,17 @@ func shouldInject(pod *corev1.Pod) bool {
 		}
 	}
 
-	return isApmInstrumentationEnabled(pod.GetNamespace()) || config.Datadog.GetBool("admission_controller.mutate_unlabelled")
+	return isApmInstrumentationEnabled() || config.Datadog.GetBool("admission_controller.mutate_unlabelled")
 }
 
 // isApmInstrumentationEnabled indicates if Single Step Instrumentation is enabled for the namespace in the cluster
```

Comment:
I think we should update the comment too. 

this function is not anymore specific to a certain namespace, it is now a global config,

Reply:
Right. Thanks :+1: